lib/util/emoji.ts

```diff
@@ -1,16 +1,101 @@
-import emoji from 'node-emoji';
+import is from '@sindresorhus/is';
+import mathiasBynensEmojiRegex from 'emoji-regex';
+import {
+  fromCodepointToUnicode,
+  fromHexcodeToCodepoint,
+  fromUnicodeToHexcode,
+} from 'emojibase';
+import emojibaseEmojiRegex from 'emojibase-regex/emoji';
+import SHORTCODE_REGEX from 'emojibase-regex/shortcode';
 import type { RenovateConfig } from '../config/types';
+import dataFiles from '../data-files.generated';
+import { regEx } from './regex';
 
 let unicodeEmoji = true;
 
+let mappingsInitialized = false;
+const shortCodesByHex = new Map<string, string>();
+const hexCodesByShort = new Map<string, string>();
+
+function lazyInitMappings(): void {
+  if (!mappingsInitialized) {
+    const table: Record<string, string | string[]> = JSON.parse(
+      dataFiles.get('node_modules/emojibase-data/en/shortcodes/github.json')
+    );
+    for (const [hex, val] of Object.entries(table)) {
+      const shortCodes: string[] = is.array<string>(val) ? val : [val];
+      shortCodesByHex.set(hex, `:${shortCodes[0]}:`);
+      shortCodes.forEach((shortCode) => {
+        hexCodesByShort.set(`:${shortCode}:`, hex);
+      });
+    }
+    mappingsInitialized = true;
+  }
+}
+
 export function setEmojiConfig(_config: RenovateConfig): void {
   unicodeEmoji = _config.unicodeEmoji;
 }
 
+const shortCodeRegex = regEx(SHORTCODE_REGEX.source, 'g');
+
 export function emojify(text: string): string {
-  return unicodeEmoji ? emoji.emojify(text) : text;
+  lazyInitMappings();
```

Comment:
```suggestion
  if (!unicodeEmoji)
    return text;
  lazyInitMappings();
```

shortcut? so skip init?